kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -98,42 +98,36 @@ public boolean strict() {
     @Parameter(names="--read-only-kompiled-directory", description="Files in the generated kompiled directory should be read-only to other frontend tools.")
     public boolean readOnlyKompiledDirectory = false;
 
-    @ParametersDelegate
-    public Experimental experimental = new Experimental();
-
     public boolean isKore() {
         return backend.equals("kore") || backend.equals("haskell") || backend.equals("llvm");
     }
 
-    public static final class Experimental implements Serializable {
-
-        @Parameter(names="--heat-cool-by-strategies", description="Control heating and cooling using strategies.")
-        public boolean heatCoolStrategies = false;
+    @Parameter(names="--heat-cool-by-strategies", description="Control heating and cooling using strategies.")
```

Comment:
This option should probably be removed.

Reply:
This is done, but note that this option was still actually used somewhere (when constructing a `Strategy`). You should double-check that I did the correct dead-code elimination by propagating the default value `false` through to its uses.